styles/_custom.css

```diff
@@ -1,6 +1,8 @@
 body {
   font-family: "Apercu";
   -webkit-font-smoothing: antialiased;
+  overflow-y: scroll; /* has to be scroll, not auto */
```

Comment:
this may be an issue with scroll-disabling further down the line but good for now!